### PR TITLE
Feature/orange dividers

### PR DESCRIPTION
**Pull request by joshuascan**
## Description

[Loom Video](https://www.loom.com/share/d29a59a59ecf4d458d99b08da08226e4)

This branch was originally to update the orange dividers in the Admin Tools section to match the orange dividers throughout the rest of the page. However, once I began working on it, I noticed that there were other sections of the website that needed the revised dividers so I changed the scope of the branch to encompass the entire site.

For reference, the old dividers were an image of a solid orange bar, whereas the new dividers are created through CSS.

This revision helps to address the "Product Polish" MVP goal and works towards implementing uniformity in style across the site.

[Trello card](https://trello.com/c/hnvoMGpG)

Fixes # (issue)

Updates old image-based dividers with CSS-based dividers throughout the site. The now unnecessary imports (of the image and ant design icon component) have also been removed from the corresponding pages.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by chrisjs87 — CHANGES_REQUESTED** [review]
There is a comment that should be deleted before this is approved.

However, the rest of my comments are really a design critique, and can be subjective to what is desired for the project. Just my 2 cents that a 1px border may not be the most aesthetically appealing, and that we might like for it to be a couple pixels bigger to match the proportion of the page. For example. the border pixels might be smaller on the add a user form, while much larger on the judges or cases page, as the line will be taking up more space, and will be more proportionally pleasing if it is larger!

File: src/components/pages/AccountPage/_AccountPageStyles.less [hunk 1]
```diff
@@ -11,6 +11,13 @@
   margin-right: 3rem;
 }
 
+.account-divider {
+  border-bottom: 1px solid #ff7f0e;
```
[the remark this answers is not in the document]
Comment (joshuascan) [reply src/components/pages/AccountPage/_AccountPageStyles.less:15]: I might agree, however, I was matching the size that had already been implemented throughout the rest of the site. The changing of the size would be more of a styling issue and would be something that we should discuss as a group, whereas this ticket was just about completing an incomplete feature update.

File: src/components/pages/JudgeTable/_JudgeTableStyles.less [hunk 2]
```diff
@@ -23,7 +28,7 @@
   padding: 1rem;
   display: flex;
   flex-direction: column;
-  border-top: 1px solid #ff7f0e;
+  //   border-top: 1px solid #ff7f0e;
```
[the remark this answers is not in the document]
Comment (joshuascan) [reply src/components/pages/JudgeTable/_JudgeTableStyles.less]: Deleted! Thank you for your clear and in-depth review. The organization makes it very easy to follow!

File: src/components/pages/AccountPage/AccountPage.js [hunk 3]
```diff
@@ -3,8 +3,6 @@ import { Button, Card, Avatar, Modal, Form, Input } from 'antd';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
 
 import './_AccountPageStyles.less';
-import Icon from '@ant-design/icons';
```
Comment (jenna-anderson) [inline src/components/pages/AccountPage/AccountPage.js:6]: Good catch on removing the unnecessary imports after the changes

File: src/components/pages/AccountPage/AccountPage.js [hunk 4]
```diff
@@ -145,13 +140,7 @@ const AccountPage = props => {
...
               >
                 <h2 className="h1Styles">Edit User</h2>
-                <p className="divider">
-                  <Icon
-                    component={() => (
-                      <img src={OrangeLine} alt="divider icon" />
-                    )}
-                  />
-                </p>
+                <div className="divider" />
```
Comment (jenna-anderson) [inline src/components/pages/AccountPage/AccountPage.js:143]: Good call on changing the p tag for the divider, especially for accessibility reasons

**Review by jenna-anderson — APPROVED** [review]
Your description and video were clear and helpful in understanding the changes, why they were made, and how it directly relates to the MVP. 
Nice attention to detail noticing other parts of the site outside of Admin Tools were also not matching and explaining how this broadened the scope of the PR while still keeping it focused on solving one problem. 
Your commits were atomic and clear. Nice work!

**Review by schandrupatla — APPROVED** [review]
Good work. Your video explanation was very neat having the web page and code side by side . Easy to understand what changes have been made and where. You have good attention to detail like how you fixed the margins and spaces to be consistent throughout the application.

File: src/components/pages/AccountPage/_AccountPageStyles.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisjs87) [reply src/components/pages/AccountPage/_AccountPageStyles.less:15]: That makes a lot of sense! I agree with your approach.

**Review by chrisjs87 — APPROVED** [review]
My comments have been addressed. Great job! You made decisions to keep changes to the app as consistent to existing patterns as possible.

File: src/components/pages/AccountPage/AccountPage.js (hunk 4 above)
Comment (paulstgermain) [reply src/components/pages/AccountPage/AccountPage.js:143]: Seconding this - really good catch here!

**Pull request merged**